cmd/certificates/setup.go

```diff
@@ -55,7 +55,11 @@ func cmdSetup(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if config.Identity.Status() != identity.CertKey {
+	status, err := config.Identity.Status()
+	if err != nil {
+		return err
+	}
+	if status != identity.CertKey {
```

Comment:
is there a way that identity.CertKey could be empty and still of that type? would that need to be checked here? (idk but just wondering. maybe that's something that could be handled somewhere else.)